src/core/task.ts

```diff
@@ -1,9 +1,10 @@
-import { AsyncTask, TaskFunction, TaskResolve, TaskTiming } from "@clarity-types/core";
+import { AsyncTask, TaskFunction, TaskResolve } from "@clarity-types/core";
 import { Metric } from "@clarity-types/data";
 import config from "@src/core/config";
 import * as metric from "@src/data/metric";
 
-let tracker: TaskTiming = {};
+let tracker: { [key: number]: number } = {};
```

Comment:
Please add a comment about wat these represent. From names and data type very hard to glean much.

Reply:
Done